src/formatPkg.ts

```diff
@@ -634,3 +637,14 @@ function getModuleTypes(pkg: NicePackageType): ModuleType[] {
 
   return [...moduleTypes];
 }
+
+function getStyleTypes(pkg: NicePackageType): StyleType[] {
```

Comment:
Is this relevant to do it at this stage if you always override the information later?
Especially if values is not filtered (cf: .woff)

Reply:
Why does it get overridden? isn't the other value packageType?

Reply:
@bodinsamuel It doesn't get overridden but updated with new values. 
Here I get the file extension from the `style` field of package.json and letter add the extensions of style files from a listing.